src/encoded/static/components/antibody.js

```diff
@@ -272,14 +282,18 @@ const AntibodyStatus = (props) => {
 
         // Look at all organisms in current lot_review. They go under this lot_review's status
         const statusNode = statusTree[lotReview.status];
+        if (!lotReview.organisms.length) {
```

Comment:
For [AirBnb](https://github.com/airbnb/javascript#comparison--shortcuts) compliance, this should be `lotReview.organisms.length === 0`.

Reply:
Updated. Thank you.